localization/yabloc/yabloc_common/src/camera_info_subscriber.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "yabloc_common/camera_info_subscriber.hpp"
+
+namespace yabloc::common
+{
+CameraInfoSubscriber::CameraInfoSubscriber(rclcpp::Node * node)
+{
+  rclcpp::QoS qos(10);
+  auto callback = [this](const CameraInfo & msg) -> void { opt_info_ = msg; };
+  sub_info_ = node->create_subscription<CameraInfo>("input/camera_info", qos, callback);
```

Comment:
```suggestion
  sub_info_ = node->create_subscription<CameraInfo>("~/input/camera_info", qos, callback);
```
https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/topic-namespaces/

Reply:
Fixed in d6c0f9a